ScnSoft-Education/LeetCodeTraining.Tests/StringToIntegerATOITests.cs

```diff
@@ -0,0 +1,20 @@
+using LeetCodeTraining.StringToIntegerATOI;
+
+namespace LeetCodeTraining.Tests
+{
+    public class StringToIntegerATOITests
+    {
+        [Theory]
```

Comment:
apply "format document" always before commit your code
In Visual studio -> Edit -> Advanced -> format document
or use hot key ctrl + k, d